### PR TITLE
Community guidelines

### PR DESCRIPTION
**Pull request by djbe**
Based on https://github.com/Moya/contributors

### CODE REVIEW

File: COMMUNITY.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+If you get a merged Pull Request, regardless of content (typos, code, doc fixes), then you're eligible for push access to this organization. Just ask, and we'll gladly give you push access, no strings attached.
+
+Offhand, it's easy to imagine that this would make code quality suffer, but in reality it offers fresh perspectives to the codebase and encourages ownership from people who are depending on the project. If you are building a project that relies on this codebase, then you probably have the skills to improve it and offer valuable feedback.
+
+Everyone comes in with their own perspective on what a project could/should look like, and encouraging discussion can help expose good ideas sooner.
+
+#### Why do we give out push access?
+
+It can be overwhelming to offered the chance to wipe the source code for a project. Don't worry, we don't let you push to master. All code is peer-reviewed, and we have the convention that someone other than the submitter should merge non-trivial pull requests.
```
Comment (AliSoftware) [inline COMMUNITY.md]: to /be/ offered

File: COMMUNITY.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+This project is not continuously deployed, there is space for debate after review too. Offering everyone the chance to revert, or make an amending pull request. If it feels right, merge. 
+
+#### How can we help you get comfortable contributing?
+
+It's normal for a first pull request to be a potential fix for a problem, and moving on from there to helping the project's direction can be difficult. We try to help contributors cross that barrier by offering good first step issues. These issues can be fixed without feeling like you're stepping on toes. Ideally, these are non-critical issues that are well defined. They will be purposely avoided by mature contributors to the project, to make space for others.
+
+We aim to keep all project discussion inside GitHub issues. This is to make sure valuable discussion is accessible via search. If you have questions about how to use the library, or how the project is running - GitHub issues are the goto tool for this project.
+
+For actualy contribution guidelines, you can always check our [contributing documentation](CONTRIBUTING.md).
```
Comment (AliSoftware) [inline COMMUNITY.md]: s/actualy/actual/

**Comment by djbe** [comment]
I just made a specific PR for this (and would like to do so for other such documents), so that we can review and merge them separately, even though they might refer to each other.

**Comment by AliSoftware** [comment]
Merged from Amsterdam 🇳🇱 (where I'm spending the weekend after AppDevCon) 🍺😀

**Pull request merged**